### ISSUE TITLE
Windows 10 Start menu item pinning - removal not working

### ISSUE THREAD

**Issue by mustafatoktas** [valinet/ExplorerPatcher#3968]
I am using the Windows 10 full screen start menu.

When I right click on the items I have pinned in the start menu and want to remove them from the start, I cannot do this. Because in the context menu that opens with a right click, it says pin to start, whereas the item is already pinned in the start menu.

Similarly, when I right click on an item that is not in the start menu and click pin to start, it does not pin it in the start menu.

In other words, I cannot add a new item to the start menu, nor can I change the existing items.

---------------
EP:  22621.4317.67.1
OS:  win 11 home 23h2 22631.4541

**Comment by Amrsatrio** [comment]
Uninstall the preview update. I'll have this fixed in the next EP version.

**Comment by pyrates999** [comment]
> I am still having this issue with the latest version

You should create a new issue as this one is closed.
